projects/uitdatabank/reference/entry.json

```diff
@@ -9496,6 +9496,320 @@
           }
         ]
       }
+    },
+    "/cultuurkuur/regions": {
+      "get": {
+        "summary": "/cultuurkuur/regions",
```

Comment:
This should be something like `regions - GET`, you can compare this with existing endpoints.